hatch/src/cli/commands/build.rs

```diff
@@ -5,6 +5,8 @@ use cli::commands::ops::ProjectOps;
 use yaml;
 use project::Project;
 
+use cli::commands::{ PROJECT_NAMES };
```

Comment:
This is a question for the style guide, if it is only one item, should we use the { }?  use ```cli::commands::PROJECT_NAMES;``` vs. ```use cli::commands::{ PROJECT_NAMES };```

Reply:
I am in favor of using ```cli::commands::PROJECT_NAMES```

Reply:
Ok so maybe we should clarify something first. This is the current state of the `use` section:

```
use HatchResult;
use clap::{ App, SubCommand, Arg, ArgMatches };
use cli::commands::Command;
use cli::commands::ops::ProjectOps;
use yaml;
use project::Project;

use cli::commands::{ PROJECT_NAMES };
```

Do we want to combine the cli::commands::* together? I intentionally didn't because the type `Command` and `PROJECT_NAMES` serve different purposes. If we combined them it would be like this:
```
use cli::commands::{ Command, PROJECT_NAMES };
```

Reply:
No.  I would rather have them separate so its easy to read through out the code.